stories/index.js

```diff
@@ -3,6 +3,7 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
 // import { action } from '@storybook/addon-actions'
 // import { linkTo } from '@storybook/addon-links'
+import 'unfetch/polyfill'
```

Comment:
Hmm I added this as part of the Card dependency:
https://github.com/microlinkhq/react-microlink/commit/3166662daaa37784b00e7a7081e7c622ce4fd911#diff-e3889dc8720303359b31912c89e0a295R4

What do you think?